<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatumStudio</class>
 <widget class="QMainWindow" name="DatumStudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Datum Studio</string>
  </property>
  <widget class="QWidget" name="Layout">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="SideBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CommandBar" name="SideHeader" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MetaBar" name="SideMetaBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MetaBox" name="SideMetaBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="Splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="handleWidth">
         <number>1</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QStackedWidget" name="Container">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QStackedWidget" name="MainStatusView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page"/>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="StatusBarLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="StatusBox" name="MainStatusBox" native="true"/>
        </item>
        <item>
         <widget class="StatusBar" name="MainStatusBar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="StatusReport" name="MainStatusReport" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="About"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <action name="Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MetaBar</class>
   <extends>QWidget</extends>
   <header>metabar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MetaBox</class>
   <extends>QWidget</extends>
   <header>metabar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommandBar</class>
   <extends>QWidget</extends>
   <header>commandbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusBar</class>
   <extends>QWidget</extends>
   <header>statusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusBox</class>
   <extends>QWidget</extends>
   <header>statusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusReport</class>
   <extends>QWidget</extends>
   <header>statusbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>triggered()</signal>
   <receiver>DatumStudio</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
